kmod/src/inode.c

```diff
@@ -1016,9 +1042,11 @@ void scoutfs_update_inode_item(struct inode *inode, struct scoutfs_lock *lock,
 {
 	struct scoutfs_inode_info *si = SCOUTFS_I(inode);
 	struct super_block *sb = inode->i_sb;
+	struct scoutfs_sb_info *sbi = SCOUTFS_SB(sb);
 	const u64 ino = scoutfs_ino(inode);
 	struct scoutfs_key key;
-	struct scoutfs_inode sinode;
+	struct scoutfs_inode_payload_wrapper wrap;
```

Comment:
Try to maintain the adorable "waterfall" of local declarations?   Throw this wrapper up above ino = ?